.gitignore

```diff
@@ -10,3 +10,4 @@
 __pycache__
 /doc/build/*
 /build/*
+suit-generator.log
```

Comment:
Will this suffice taking into account rotating character of logfile?

Reply:
Improved!